DAQ/test/analyzeTrkFragments.fcl

```diff
@@ -0,0 +1,66 @@
+# Run StrawAndCaloDigisFromFragments module to convert artdaq::Fragment collections
+# into TRK  collection and analzye it
+# Usage: mu2e -c DAQ/test/analyzeTrkFragments.fcl -s <input root files> -n '-1'
+#
+#
+# Contact person Gianantonio Pezzullo
+
+#include "fcl/minimalMessageService.fcl"
+#include "fcl/standardServices.fcl"
+
+services : @local::Services.Reco
+
+# timing information
+services.TimeTracker : {
+    dbOutput : {
+	filename : ""
+	overwrite : true
+    }
+}
+
+services.scheduler.wantSummary: true
+
+process_name : analyzeTrkFragments
+
+source : {
+  module_type : RootInput
+  fileNames   : @nil
+
+  # firstEvent  : 500
+  maxEvents   : -1
+}
+
+physics : {
+
+  producers : {
+      makeSD:
+      {
+	  module_type: StrawAndCaloDigisFromFragments
+	  diagLevel: 0
+	  parseCAL: 0
+	  parseTRK: 1
+	  
+	  trkTag  : "daq:trk"
+	  caloTag : "daq:calo"
+      }
```

Comment:
Can you use pre-defined configurations of makeSD and readSD here?  Or use them with one or two overrides?  Copying it in full makes it fragile to new parameters added to the standard configuration